### ISSUE TITLE
Font awesome as icons inside a shape (circularImage)

### ISSUE THREAD

**Issue by zigmok** [visjs/vis-network#1508]
Is there any way to mix them both (shapes and icons)?  Basically I have managed to use images of fontawesome with circularImage, but gets too slow when using multiple nodes.  Is there any way to have the same effect as with circularImage but adding the icon instead of the image?  Maybe with ctxRenderer?
Regards!

**Comment by neo6053** [comment]
Yes, you can. Check icon square : https://visjs.github.io/vis-network/examples/network/nodeStyles/shapes.html

**Comment by nook24** [comment]
You can, how ever in my case, vis uses the `regular` font by default which is not available for all Font Awesome icons. So some icons are working combined with a shape and some are not.
I was not able to resolve this for now, but have descried the issue in detail in this comment: https://github.com/visjs/vis-network/issues/1967#issuecomment-2379657020

This is also broken on the vis demo @neo6053 mentioned
![grafik](https://github.com/user-attachments/assets/d1e3a4ba-509d-48f0-b20b-a5c4df30c3f7)
